src/frontend/qt_sdl/Window.cpp

```diff
@@ -753,12 +777,16 @@ MainWindow::MainWindow(int id, EmuInstance* inst, QWidget* parent) :
         actScreenRotation[windowCfg.GetInt("ScreenRotation")]->setChecked(true);
 
         int screenGap = windowCfg.GetInt("ScreenGap");
-        for (int i = 0; i < 6; i++)
+
+        if (!actScreenGap[6]->isChecked())
```

Comment:
When is `actScreenGap[6]` pre-set to checked here? Feels to me like the below `for` loop should instead be adjusted so that if none of the default gap sizes are checked, the "Custom" checkbox is checked.